### PR TITLE
added security to webhooks

### PR DESCRIPTION
**Pull request by edwardkoval**

### CODE REVIEW

File: api-reference/beta/resources/subscription.md [hunk 1]
```diff
@@ -8,6 +8,7 @@ A subscription allows a client app to receive notifications about changes to dat
 - Conversations from Office Groups.
 - Drive root items from OneDrive.
 - Users and Groups from Azure Active Directory.
+- Security API
```
Comment (preetikr) [inline api-reference/beta/resources/subscription.md]: Rephrase this as "Alerts from Graph Security"

File: api-reference/beta/resources/subscription.md [hunk 2]
```diff
@@ -56,6 +57,7 @@ Here is a JSON representation of the resource.
 | Contacts            | 4230 minutes (under 3 days)    |
 | Group conversations | 4230 minutes (under 3 days)    |
 | Drive root items    | 4230 minutes (under 3 days)    |
+| Security alerts     | 43200 minutes (under 30 days)  |
```
Comment (preetikr) [inline api-reference/beta/resources/subscription.md]: is this accurate? the numbers do not align with abovementioned numbers...

File: api-reference/beta/resources/webhooks.md [hunk 3]
```diff
@@ -14,16 +14,17 @@ Using the Microsoft Graph API, an app can subscribe to changes on the following
...
+- Security alerts
 
 ## Permissions
 
 In general, subscription operations require read permission to the resource. For example, to get notifications for messages, your app needs the `Mail.Read` permission. The [create subscription](../api/subscription_post_subscriptions.md) article lists permissions needed for each resource type. The following table lists the types of permissions your app can request to use webhooks for specific resource types.
 
 | Permission type                        | Supported resource types in beta                                                      |
 | :------------------------------------- | :------------------------------------------------------------------------------------ |
-| Delegated - work or school account     | [contact][], [conversation][], [drive][], [event][], [group][], [message][], [user][] |
+| Delegated - work or school account     | [contact][], [conversation][], [drive][], [event][], [group][], [message][], [user][], [security][] |
```
Comment (preetikr) [inline api-reference/beta/resources/webhooks.md]: not sure of security fits in this context? Why did you include security [] here?

File: api-reference/v1.0/resources/subscription.md [hunk 4]
```diff
@@ -6,6 +6,7 @@ A subscription allows a client app to receive notifications about changes to dat
 - Conversations from Office Groups
 - Drive root items from OneDrive
 - Users and Groups from Azure Active Directory
+- Security API
```
Comment (preetikr) [inline api-reference/v1.0/resources/subscription.md]: Rephrase this as Alerts from Graph Security

File: api-reference/v1.0/resources/subscription.md [hunk 5]
```diff
@@ -67,6 +68,7 @@ Here is a JSON representation of the resource.
 | Contacts            | 4230 minutes (under 3 days)    |
 | Group conversations | 4230 minutes (under 3 days)    |
 | Drive root items    | 4230 minutes (under 3 days)    |
+| Security alerts     | 43200 minutes (under 30 days)  |
```
Comment (preetikr) [inline api-reference/v1.0/resources/subscription.md]: is the number accurate (similar to earlier comment I had on this)?

File: api-reference/v1.0/resources/webhooks.md [hunk 6]
```diff
@@ -12,16 +12,17 @@ Using the Microsoft Graph API, an app can subscribe to changes on the following
...
+- Security alerts
 
 ## Permissions
 
 In general, subscription operations require read permission to the resource. For example, to get notifications for messages, your app needs the `Mail.Read` permission. The [create subscription](../api/subscription_post_subscriptions.md) article lists permissions needed for each resource type. The following table lists the types of permissions your app can request to use webhooks for specific resource types.
 
 | Permission type                        | Supported resource types in v1.0                                 |
 | :------------------------------------- | :--------------------------------------------------------------- |
-| Delegated - work or school account     | [contact][], [conversation][], [drive][], [event][], [message][] |
+| Delegated - work or school account     | [contact][], [conversation][], [drive][], [event][], [message][], [security][] |
```
Comment (preetikr) [inline api-reference/v1.0/resources/webhooks.md]: Does this apply here - similar to earlier comment I had?

File: api-reference/beta/resources/subscription.md (hunk 2 above)
Comment (edwardkoval) [reply api-reference/beta/resources/subscription.md]: yes that's what Shaofeng said

File: api-reference/beta/resources/subscription.md [hunk 7]
```diff
@@ -8,7 +8,7 @@ A subscription allows a client app to receive notifications about changes to dat
 - Conversations from Office Groups.
 - Drive root items from OneDrive.
 - Users and Groups from Azure Active Directory.
-- Security API
+- Alerts from Graph Security
```
Comment (Lauragra) [inline api-reference/beta/resources/subscription.md]: Global: Revise as "Alerts from the Microsoft Graph Security API".
